default/main.py

```diff
@@ -50,15 +59,24 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         headers={'x-apikey': user.apikey},
                         ssl=ssl_context
                     ) as resp:
-                    json = await resp.json()
-                    json['api_key'] = user.apikey
+                    js = {'api_key': user.apikey}
+                    js.update(await resp.json())
+                    
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
-                        post_response = await post.text()
-                        if post.status != 200:
-                            error_msg = 'Post to webhook failed with {}: {}'.format(post.status, post_response)
-                            logging.error(error_msg)
-                            raise HTTPException(400, 'Bad request')
+
+                    async with httpSession.post('https://webhook-dot-virustotal-step-2020.ew.r.appspot.com/',
+                                                json={'access_key': get_secret('access_key'), 'vt_key': user.apikey}, ssl=ssl_context) as auth_res:
+                        js['jwt_token'] = await auth_res.text()
+                        js['jwt_token'] = js['jwt_token'][1:-1]  # I have no idea why does webhook send a key with quotes
```

Comment:
```suggestion
                        js['jwt_token'] = js['jwt_token'].strip('"')  # I have no idea why does webhook send a key with quotes
```

This way it won't fail in case there's no quotes:
```python
>>> '"test"'.strip('"')
'test'
>>> 'test'.strip('"')
'test'
```